docs/contents/getting_started/index.md

```diff
@@ -60,6 +60,9 @@ Please refer to [this wiki page](https://github.com/mozilla/pdf.js/wiki/Frequent
     To get a local copy of the current code, clone it using git:
     <pre><code>$ git clone https://github.com/mozilla/pdf.js.git
 $ cd pdf.js
+$ npm install -g gulp-cli
+$ npm install   # <-- Add this line before running gulp
+$ gulp server
```

Comment:
First of all, we no longer recommend installing `gulp-cli` globally.
Secondly, `gulp server` does not belong under a section called "Source" which explains how to download the source-code; that's already (correctly) included under https://mozilla.github.io/pdf.js/getting_started/#trying-the-viewer

```suggestion
$ npm install
```